TMS.Net07.Homework.Algoritms/TMS.Net07.Homework.Algoritms - factorial cycle/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMS.Net07.Homework.Algoritms___factorial_cycle
+{
+    class Program
+    {
+        const string errorMessage = "Please, check your input";
+        static void Main(string[] args)
+        {
+            while (true) 
+            {
+                int index;
+                string input = Console.ReadLine();
+                //check input value
+                if (input == "exit") 
+                {
+                    return;
+                }
+                bool isOk = Int32.TryParse(input, out index);
+                if(!isOk) 
+                {
+                    Console.WriteLine($"{errorMessage}");
+                }
+                else if (index == 0)
+                {
+                    Console.WriteLine("1");
+                }
+                else if (index < 0)
+                {
+                    Console.WriteLine($"{errorMessage}");
+                }
+                else 
+                {
+                    int result = resultValue(index);
+                    Console.WriteLine($"{Environment.NewLine}{result}");
+                }
+            }
+        }
+        //method for factorial
+        static int resultValue (int index)
```

Comment:
```suggestion
        static int ResultValue (int index)
```
We are using Pascal case for method names. Camel case only for local variables.
https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841

Reply:
Done. New commit added.